tests/.coveragerc

```diff
@@ -1,5 +1,7 @@
 [run]
 branch = True
+concurrency = multiprocessing
+data_file = .coverages/.coverage
```

Comment:
why configure `data_file`? the default is normally sufficient and I can't see why not here

Reply:
I customized the value of `data_file` for a better cleanup of the workspace when using `coverage` in parallel.

When used in parallel `coverage` creates a file for each process in the path where it runs.

These files are listed _(eg: in your editor)_ even if git-ignored until they are combined and sometimes you have to delete them by hand if something goes wrong.
```
.coverage.net.24830.761944
.coverage.net.24990.580886
.coverage.net.24991.642706
.coverage.net.24992.475125
.coverage.net.24993.937970
.coverage.net.24994.804635
.coverage.net.24995.747769
.coverage.net.24996.767985
```

Using the `.coverages/.coverage` value in `data_file` causes all intermediate files to be created inside the `.coverages` directory.

Now you will see _(eg: in your editor)_ only one git-ignored `.coverages` directory and if needed you can delete all old files left by deleting the whole directory in one go.
```bash
.coverages/
```

Reply:
Okay fair enough!